### PR TITLE
Add some more books with a minor typo fix

### PR DESCRIPTION
**Pull request by greenbagels**

### CODE REVIEW

File: resources/books.md [hunk 1]
```diff
@@ -15,3 +15,12 @@ Other book lists: [freenode ##math](https://freenode-math.fandom.com/wiki/Book_L
...
   - mostly elementary proofs from various areas that are just beautiful
+
+### History
+0. John Stillwell, [Mathematics and Its History](https://www.springer.com/gp/book/9781441960528).
+  - broadly describes the development of different branches of math, with short profiles of mathematicians and optional exercises
+
+### Real Analysis
+#### Undergraduate
+0. Charles Chapman Pugh, [Real Mathematical Analysis](https://www.springer.com/gp/book/9783319177700).
+  - comprehensive text with a large variety of problems; has many pictures, multiple constructions of R, metric topology, and extra sections on linear algebra, differential forms, and lebesgue measure.
```
Comment (int-e) [inline resources/books.md]: typo: Lebesgue is a name, should be capitalized

File: resources/books.md [hunk 2]
```diff
@@ -8,10 +8,10 @@ permalink: /resources/books/
 
 This page lists books that people from Libera ##math found useful.
 
-Additions are welcome! See the [community page](/community/#contrubuting).
+Additions are welcome! See the [community page](/community/#contributing).
```
Comment (int-e) [inline resources/books.md:11]: Nice catch!

**Review by int-e — COMMENTED** [review]
Thanks, looks good to me except for one typo.

File: resources/books.md (hunk 1 above)
Comment (greenbagels) [reply resources/books.md]: Thanks for the tip; this should be fixed as of c488bc01757f1a32002b4a692509376c0f52cf61.

**Pull request merged**